http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorActor.scala

```diff
@@ -201,9 +201,9 @@ class ClusterMonitorActor(
         if (cluster.status == Starting) {
           for {
             _ <- persistInstances(cluster, googleInstances)
-            _ <- clusterHelper.stopCluster(cluster).unsafeToFuture()
-            now <- IO(Instant.now).unsafeToFuture()
-            _ <- dbRef.inTransaction { _.clusterQuery.setToStopping(cluster.id, now) }
+            _ <- clusterHelper.stopCluster(cluster)
+            now <- IO(Instant.now)
+            _ <- dbRef.inTransaction { dbRef.dataAccess.clusterQuery.setToStopping(cluster.id, now) }
```

Comment:
For classes that do a lot of DB stuff we could store the clusterQuery up front:
```
val clusterQuery = dbRef.dataAccess.clusterQuery
```
then later:
```
dbRef.inTransaction { clusterQuery.setToStopping(…) }
```
Just thinking out loud about ways to reduce syntax..